about.html

```diff
@@ -12,62 +13,98 @@
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+  <div class="container about-page">
+    <header>
+      <figure class="logo">
+        <img src="img/lambda-black.png" alt="Logo Lambda">
+      </figure>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+
+
+      </nav>
+      <figure class="jumbotron">
+        <img src="img/jumbo-about.png" alt="Jumbotron image">
+
+      </figure>
+    </header>
+    <section class="about-intro">
+      <div>
+        <h2> About</h2>
+
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews.Iterative approaches to corporate
+          strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic
+          world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+    </section>
+
+    <section class="about-content">
+
+      <section class="top-content">
+        <div class="text-container">
+          <figure class="aboutplan">
+            <img src="img/about-plan.png" alt="strategy">
+          </figure>
+          <h2> Strategy</h2>
+          <p>
+            Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+        </div>
+        <div class="text-container">
+          <figure class="aboutworking">
+            <img src="img/about-working.png" alt="strategy">
```

Comment:
Remember to change the alt attribute to reflect the description of the image. Same for lines 70 and 79.

Reply:
ok